packages/babel-traverse/src/scope/index.js

```diff
@@ -73,20 +73,21 @@ let collectorVisitor = {
   },
 
   ExportDeclaration: {
-    exit({ node, scope }) {
+    exit(path) {
+      const { node, scope } = path;
       let declar = node.declaration;
       if (t.isClassDeclaration(declar) || t.isFunctionDeclaration(declar)) {
         let id = declar.id;
         if (!id) return;
 
         let binding = scope.getBinding(id.name);
-        if (binding) binding.reference();
+        if (binding) binding.reference(path);
```

Comment:
I'm just not sure if it should be `binding.path` or `path` ... I guess it should be `path`, or both path and binding.path...